nuxeo-runtime/nuxeo-runtime-stream/src/test/resources/data/example.json

```diff
@@ -0,0 +1,9 @@
+{"namespace": "example.avro",
+ "type": "record",
+ "name": "User",
+ "fields": [
+     {"name": "name", "type": "string"},
+     {"name": "favorite_number",  "type": ["int", "null"]},
+     {"name": "favorite_color", "type": ["string", "null"]}
+ ]
+}
```

Comment:
Put an extra line at the end.

Reply:
Done, github seems not to detect it but you can see it in the files changed tab